### PR TITLE
Geocoder: remove <i> tags.

### PR DESCRIPTION
**Pull request by nadine-piveteau**
Goal: Remove <i> tags in "geocoding results" info-box.

[link](https://codepen.io/geoadmin/pen/JGgzQG?editors=0010)

### CODE REVIEW

File: src/ga/map.js [hunk 1]
```diff
@@ -244,21 +244,22 @@ ga.Map.prototype.handleHighlightError_ = function() {
...
 ga.Map.prototype.showGeocoderDialog_ = function(results) {
   this.geocoderDialog_.setSafeHtmlContent(goog.html.SafeHtml.create('div', {'id': 'geocoderList'}));
-  this.geocoderDialog_.setVisible(true); 
+  this.geocoderDialog_.setVisible(true);
   this.geocoderList_ = new goog.ui.Menu();
   var geocoderListContainer = goog.dom.getElement('geocoderList');
   for (var item in results) {
      this.geocoderList_.addChild(
        new goog.ui.MenuItem(results[item]['attrs']['label'].
         replace('<b>','').replace('</b>',''),
```
Comment (oterral) [inline src/ga/map.js]: Replace the comma by a point at the end of the line

File: src/ga/map.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ goog.require('goog.ui.Menu');
 goog.require('goog.ui.MenuItem');
 goog.require('goog.dom');
 goog.require('goog.dom.classlist');
+goog.require('goog.html.SafeHtml');
```
Comment (oterral) [inline src/ga/map.js]: Does it works without this require ?

Comment (nadine-piveteau) [reply src/ga/map.js]: No it does not. When I run the command `make -f Makefile-ga build-ga` I get the following error:
`
ERR! compile /home/ltpin/geoadmin-ol3/src/ga/map.js:246: ERROR - missing require: 
'goog.html.SafeHtml'
ERR! compile   this.geocoderDialog_.setSafeHtmlContent(goog.html.SafeHtml.create('div', {'id': 'geocoderList'}));
ERR! compile                                         
ERR! compile
ERR! compile 1 error(s), 0 warning(s)
ERR! compile
ERR! Process exited with non-zero status, see log for more detail: 1
Makefile-ga:14: recipe for target 'build/ga.js' failed
`

**Comment by oterral** [comment]
Thx

**Pull request merged**